### ISSUE TITLE
Can we have labels to better categorize issues?

### ISSUE THREAD

**Issue by neauoire** [dariusk/NaNoGenMo-2015#37]
I will make something for this.
Shouldn't we all commit on this repo as branches or something?
Instead of issues?

**Comment by dariusk** [comment]
Nope, I'm using issues as a forum system. You don't even need to use Github
if you don't want to for your code. Just link a novel and your code from
here, whether that's on Github or somewhere else. I'm trying to keep this
as tech/platform neutral as possible.

**Comment by neauoire** [comment]
Gotcha,
Could we suggest using tags/labels to distinguish between intent or ideas to keep issues in order.
Or a sort of tenplate to make easy for everyone to browse issues?

example: [40%] Story Name - Ruby

**Comment by enkiv2** [comment]
Tags are typically used to indicate milestones -- if a novel snapshot has
been posted, for instance.

**Comment by hugovk** [comment]
Last year we had `preview` and `completed` labels:

https://github.com/dariusk/NaNoGenMo-2014/labels

**Comment by neauoire** [comment]
That could work : )

**Comment by dariusk** [comment]
I'm not sure we can realistically enforce that... also percentages don't
make sense. I can generate 50k words in five seconds. It's the quality that
counts and that's nonlinear.

I do want to encourage people to title their issues something descriptive,
though most people don't know what they're going to make yet.

**Comment by dariusk** [comment]
I think @hugovk and I could simply ask people who seem to have an idea
already to update their issue title accordingly.

**Comment by hugovk** [comment]
I also think labels are useful, especially for completed entries, because I can quickly go and see the full list from last year and also get a quick count (91 completed, 12 previews).

**Comment by dariusk** [comment]
Oh I agree we need labels! I can't think of more useful labels than the ones we already have, though of course I'm open to ideas.

**Comment by neauoire** [comment]
I don't seem to be able to pick one for my issue, is there a sort of privacy setting on here?

**Comment by dariusk** [comment]
Only admins can tag. (Maybe we should open that up?)

**Comment by enkiv2** [comment]
Could we have a tag for "source available"? Often, projects have really
interesting behavior but aren't considered to be producing sufficiently
interesting material for a novel-length work, and it's easy to miss when we
only mark based on what the author considers 'complete'.

**Comment by hugovk** [comment]
@enkiv2:

> The only rule is that you share at least one novel and also your source code at the end.

No source = not complete.

**Comment by dariusk** [comment]
I see John's point though. There may be an incomplete, non-novel that still
has source code of note. I'll add that tag.

**Comment by dariusk** [comment]
@neauoire I've added an admin label to this issue since it's meta-discussion. You might want to open a new issue for your own participation in the event.
